torch/_inductor/codecache.py

```diff
@@ -2050,8 +2117,16 @@ def load_async(cls, source_code: str, cuda=False, submit_fn=None):
             "cuda": cuda,
             "vec_isa": pick_vec_isa(),
         }
-        cpp_command = repr(cpp_compile_command("i", "o", **compile_command))
-        key, input_path = write(source_code, "cpp", extra=cpp_command)
+
+        from torch._inductor.jit_builder.cpp_builder import CppBuilder, CppTorchOptions
```

Comment:
why not importing it globally?

Reply:
1. codecache.py will import cpp_builder's CppBuilder.
2. cpp_builder.py CppBuilder will import codecache's VecISA.
If both 1 and 2 in global scope, it would occurred `circulate import` issue when initial time.
So, It is workaround to avoid issue.

Reply:
Another optimization is move the VecISA to jit_builder.py after all PR landed.

Reply:
Yes, moving the isa detection/check logic into a separate module sounds more reasonable (maybe not the builder module but separate) since it can be used by other modules too.

Reply:
Maybe move it to a separate VecISA.py is better.